src/mlbstandings/web.py

```
@@ -7,21 +7,31 @@
 class Web:
     @staticmethod
     def retryable(code: int) -> bool:
-        return False  # TODO
+        return code in set([429, 500, 503])
+    @staticmethod
```

Flake8 (7.2.0)

[error] expected 1 blank line, found 0

(E301)